src/main/java/com/wcc/platform/service/CmsService.java

```diff
@@ -16,86 +16,84 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
-/**
- * CMS service responsible for simple pages.
- */
+/** CMS service responsible for simple pages. */
 @Service
 public class CmsService {
 
-    private final ObjectMapper objectMapper;
+  private final ObjectMapper objectMapper;
 
-    @Autowired
-    public CmsService(final ObjectMapper objectMapper) {
-        this.objectMapper = objectMapper;
-    }
+  @Autowired
+  public CmsService(final ObjectMapper objectMapper) {
+    this.objectMapper = objectMapper;
+  }
 
-    /**
-     * Read JSON and convert to Pojo TeamPage.
-     *
-     * @return Pojo TeamPage.
-     */
-    public TeamPage getTeam() {
-        try {
-            final File file = Path.of(FileUtil.getFileUri(TEAM.getFileName())).toFile();
-            return objectMapper.readValue(file, TeamPage.class);
-        } catch (IOException e) {
-            throw new PlatformInternalException(e.getMessage(), e);
-        }
+  /**
+   * Read JSON and convert to Pojo TeamPage.
+   *
+   * @return Pojo TeamPage.
+   */
+  public TeamPage getTeam() {
```

Comment:
please revert this changes, because they have to be as string for now because of docker limitiation.. unless you manage to fix that. I mentioned in the deployment PR that getFileUri does not work on docker deployment